.gitignore

```diff
@@ -0,0 +1,5 @@
+.terraform.lock.hcl
```

Comment:
.gitignore is super super important.
If we want to help our fellow developers not to push secret ssh keys to git so instead hardcoding your secret name `my_key.pem` we can use wildcard -->
`*.pem`

Reply:
got it!